.github/workflows/tests.yml

```diff
@@ -1,38 +1,31 @@
 name: tests ci
-# This workflow runs the pipeline with the minimal test dataset to check that it completes any errors
+# This workflow runs the pipeline with the minimal test dataset to check 
+# is completed without any errors
 on:
-  push:
-    branches: [develop]
-  pull_request_target:
-    branches: [develop]
-  release:
-    types: [published]
+    pull_request:
+    push:
 
 jobs:
-  push_dockerhub:
-    name: Push new Docker image to Docker Hub (dev)
+  create-conda-env:
```

Comment:
```suggestion
  tests:
```

Reply:
test file updated to use the pyroject.toml file instead of setup.py 
```
- name: Activate env and install taranis
        run: |
          source $CONDA/etc/profile.d/conda.sh
          conda activate taranis_env
          poetry install
          taranis analyze-schema -i  test/MLST_listeria -o analyze_schema_test  --cpus 1 --output-allele-annot --remove-no-cds --remove-duplicated --remove-subset
```